### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by GNOCCHIMOBBYN** [TannerGabriel/discord-bot#99] [General]
Is there possible to add forwarding and rewinding commands?

**Comment by TannerGabriel** [comment]
I personally have never thought of that. It would probably be possible, but I think that it would be hard to implement. That said, I might find the time to research it a bit more and find a way (but no promises).

**Reply by Arrong18** [reply]
There are ways to implement a seek command which would restart playing the song at a certain point but I think essentially it calls the play command but with a time stamp attached. I do have code for this if you want it but might need changing a little as it's a little buggy at times (slow).

**Reply by TannerGabriel** [reply]
There is already a PR for the seek command #145.
